pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -235,3 +269,74 @@ func StripRootFSMountPoint(rootfsMount string) *metadataOptions {
 		},
 	}
 }
+
+// durationToHumanReadableString formats a duration into a human-readable string.
+// Unlike Go's built-in `time.Duration.String()`, which returns a string like "72h0m0s", this function returns a more concise format like "3d" or "5d4h25m".
+func durationToHumanReadableString(d time.Duration) string {
```

Comment:
could we include a link to to original implementation and state that we slightly changed the code ?